api/config/packages/dev/lexik_jwt_authentication.yaml

```diff
@@ -0,0 +1,10 @@
+parameters:
+    env(COOKIE_PREFIX): ""
+
+lexik_jwt_authentication:
+    set_cookies:
+        '%env(COOKIE_PREFIX)%jwt_hp':
+            secure: false
+        
+        '%env(COOKIE_PREFIX)%jwt_s':
+            secure: false
```

Comment:
What is the problem here?

Reply:
Safari seems to not set the cookie on http://localhost, if it's marked as `secure`.

Chrome is ignoring the `secure` config on localhost (which is the expected behavior).